### PR TITLE
Updated to align with the ISO standards

### PR DESCRIPTION
**Pull request by lalc**
Please review

### CODE REVIEW

**Review by georgejpadayatti — APPROVED** [review]

File: interface-specs/data-agreement-schema/v1/data-agreement-schema.json [hunk 1]
```diff
@@ -1,57 +1,52 @@
...
-      "attribute_category": "General"
+      "attribute_name": "Age"
     }
   ],
 
   "dpia": {
-    "dpia_done": true,
     "dpia_date": "2021-05-08T08:41:59+0000",
-    "dpia_verification_url": "https://org.com/dpia_results.html"
+    "dpia_summary_url": "https://org.com/dpia_results.html"
```
Comment (jose-gataca) [inline interface-specs/data-agreement-schema/v1/data-agreement-schema.json:37]: Is this better than updating on the status of a dpia?
like "dpia_status": "NOT_STARTED | EXECUTED | APPROVED | TO REWORK" or similar

File: interface-specs/data-agreement-schema/v1/data-agreement-schema.json [hunk 2]
```diff
@@ -1,57 +1,52 @@
...
   },
 
-  "sender_metadata": {
-    "sender-did": "did:mydata:0:<sender_did_value>",
-    "preparation-time-stamp": "2021-05-08T08:41:59+0000",
-    "prepartion_signature": "<signature>"
+  "event":[{
+    "time-stamp": "2021-05-08T08:41:59+0000",
+    "principle-did": "did:mydata:1:<sender_did_value>",
+    "state": "prepared",
```
Comment (jose-gataca) [inline interface-specs/data-agreement-schema/v1/data-agreement-schema.json:43]: Can you elaborate on the state posible values?

File: interface-specs/data-agreement-schema/v1/data-agreement-schema.json (hunk 1 above)
Comment (jose-gataca) [reply interface-specs/data-agreement-schema/v1/data-agreement-schema.json:37]: If it's not needed, why not only display a reference to the summary, which would also include the date? Simplify the object for a URI reference

**Review by jose-gataca — CHANGES_REQUESTED** [review]
I would be missing a reference somewhere to the pii_subject.

Also, I would suggest using linked data proofs or similar: https://w3c-ccg.github.io/ld-proofs/#proof-algorithm. I think that would ease the adaptation and integration, as well as understanding the cryptographic suites and protocols to be able to perform those signatures.

Finally, I am not sure if I understand what should happen with the PRs 6 and 7 of both Jan and me.

**Comment by lalc** [comment]
> I would be missing a reference somewhere to the pii_subject.

It's as in the DA spec. pii-subject comes in during the capture phase. 
> 
> Also, I would suggest using linked data proofs or similar: https://w3c-ccg.github.io/ld-proofs/#proof-algorithm. I think that would ease the adaptation and integration, as well as understanding the cryptographic suites and protocols to be able to perform those signatures.

Thanks. Will take care of this in the signature. Good point.

> 
> Finally, I am not sure if I understand what should happen with the PRs 6 and 7 of both Jan and me.

Your PR for adding vui-implementation can remain once you fix as discussed.

When it comes to updated the data-agreement schema, this will hold and my assumption was you have raised the PR for discussions.

File: interface-specs/data-agreement-schema/v1/data-agreement-schema.json (hunk 1 above)
Comment (lalc) [reply interface-specs/data-agreement-schema/v1/data-agreement-schema.json:37]: Lets discuss this during the call

File: interface-specs/data-agreement-schema/v1/data-agreement-schema.json (hunk 2 above)
Comment (lalc) [reply interface-specs/data-agreement-schema/v1/data-agreement-schema.json:43]: I have corrected the possible values based on the DA workflow. The possible values are <Defintion/Prepration/Capture/Proof>

**Review by georgejpadayatti — APPROVED** [review]

**Comment by JanLin** [comment]
Recommend make some changes to the schema.

1. Change field "data_retention_period_in_months" to "data_retention". The field name should include format of data. This shall be done descriptively.
2. Jurisdiction example should simply be country. Remove Stockholm so it only reads "Sweden".
3. The field "geographic_restriction" has example "Europe, Not restricted" which makes it unclear. The field when included is automatically a restriction. Instead includes basis for restriction which may be storage_location or jurisdiction. So either "Europe" or "Sweden".
4. Add under "Name" category personal data a field indicating it is sensitive:

"attribute_sensitive": "True"

**Comment by lalc** [comment]
> Recommend make some changes to the schema.
> 
> 1. Change field "data_retention_period_in_months" to "data_retention". The field name should include format of data. This shall be done descriptively.
Done
> 2. Jurisdiction example should simply be country. Remove Stockholm so it only reads "Sweden".
Done
> 3. The field "geographic_restriction" has example "Europe, Not restricted" which makes it unclear. The field when included is automatically a restriction. Instead includes basis for restriction which may be storage_location or jurisdiction. So either "Europe" or "Sweden".
Done
> 4. Add under "Name" category personal data a field indicating it is sensitive:
> "attribute_sensitive": "True"
We had removed this earlier based on the discussion. Added this back now.

**Review by JanLin — APPROVED** [review]

**Review by jose-gataca — APPROVED** [review]

**Pull request merged**